packages/user-access-policy/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import createImageCaptchaConfigResolver from "./config/imageCaptcha/resolver/createImageCaptchaConfigResolver.js";
+import createRequestInspector from "./requestInspector/createRequestInspector.js";
+import type RequestInspector from "./requestInspector/requestInspector.js";
+import createExpressRuleRouter from "./rule/api/createExpressRuleRouter.js";
```

Comment:
Please move imports underneath license. biome messes this up unless you leave a space.